src/configure/configure.ts

```diff
@@ -278,17 +278,17 @@ class Orchestrator {
         return this.inputs.potentialTemplates[0].targetType;
     }
 
-    private async analyzeNode(node: any): Promise<void> {
+    private async analyzeNode(node: IResourceNode | vscode.Uri): Promise<void> {
         if (!!node) {
-            if (await this.extractAzureResourceFromNode(node)) {
+            const folderNode = node as vscode.Uri;
+            if (folderNode && folderNode.fsPath) {
```

Comment:
ohh, it worked! what is the different compared to what you tried during call?

Reply:
``` node as vscode.Uri ``` will still be required and then if() will just check fspath.